src/app/(app)/leaves/actions/get-leave-request-by-date.ts

```diff
@@ -0,0 +1,23 @@
+import { Tables } from "@/lib/db";
+import { getSupabaseClient } from "@/utils/supabase/supabaseClient";
+
+export const getleaves = async () => {
+  const supabase = await getSupabaseClient();
+
+  // Get the current month's start and end dates
+  const now = new Date();
+  const startOfMonth = new Date(now.getFullYear(), now.getMonth(), 1).toISOString();
+  const endOfMonth = new Date(now.getFullYear(), now.getMonth() + 1, 0, 23, 59, 59, 999).toISOString();
```

Comment:
@ghulamhafeez  instead of doing these calculation we should use `moment` library

Reply:
Fixed